src/flag_gems/ops/gelu.py

```diff
@@ -31,9 +32,56 @@ def gelu_tanh(x):
     return output
 
 
+@pointwise_dynamic(promotion_methods=[(0, 1, "DEFAULT")])
+@triton.jit
+def gelu_backward_none(x, dy):
+    scale1: tl.constexpr = 0.7071067811
+    scale2: tl.constexpr = 0.3989422803
+    x_fp32 = x.to(tl.float32)
+    dydx = (
+        scale2 * x_fp32 * exp(-pow(scale1 * x_fp32, 2))
+        + 0.5 * erf(scale1 * x_fp32)
+        + 0.5
+    )
+    dx = dydx * dy
+    return dx
+
+
+@pointwise_dynamic(promotion_methods=[(0, 1, "DEFAULT")])
+@triton.jit
+def gelu_backward_tanh(x, dy):
+    x_fp32 = x.to(tl.float32)
+    tanh_out = tanh(0.79788456 * x * (1 + 0.044715 * pow(x_fp32, 2)))
```

Comment:
Add some comment on the constant

`0.79788456 # math.sqrt(2 / math.pi)`

Reply:
ok. I used sympy package to help derive the derivatives, so for the other constant "0.1070322243", I actually didn't try to understand where it comes from. :(